pattoo_shared/phttp.py

```diff
@@ -321,6 +693,78 @@ def purge(url, identifier):
                 log.log2info(1007, log_message)
 
 
+def encrypted_purge(gpg, symmetric_key, req_session,
```

Comment:
If the basic action of purging is the same for encrypted and unencrypted files why repeat it? not have a shared function that does the deletion and make purge and encrypted_purge call it, then do the post? Or possibly create a Purge class because there are shared things that need to be done after the post too (This could be complicated as you'll want the class instantiated with the same parameters, which may not be possible)